tools/codestyle/ms-rebase-style.sh

```diff
@@ -0,0 +1,130 @@
+#!/bin/bash
```

Comment:
Better `!#/usr/bin/env bash`, if bash is really needed, `#!/bin/sh` otherwise

Reply:
By using <code>#!/usr/ben/env bash</code> you certainly know you get <code>bash</code>. I see some system were <code>/bin/sh</code> is the good old Bourne Shell. Not sure the script will run in this shell too.

Reply:
As noted later this script indeed needs bash, Bourne Shell won't do

Reply:
Thanks, fixed